### PR TITLE
Animation of the evolution of the temperature

### PR DESCRIPTION
**Pull request by EthanCabon**
Creation of a gif for the evolution of the temperature from time 0 to the end of the cooling. There are still mistakes about the size of the layers in the animation. Creation of the document requirements.txt.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+yaml
```
Comment (ilyasst) [inline requirements.txt]: The name of this package is actually `pyyaml` (https://pypi.org/project/PyYAML/), could you please change it ?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+os.path
```
Comment (ilyasst) [inline requirements.txt]: `os` is included by default with python and does not require to be installed using pip or conda. Please remove it.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+numpy
+math
```
Comment (ilyasst) [inline requirements.txt]: `math` is included by default with python and does not require to be installed using pip or conda. Please remove it.

File: requirements.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+matplotlib.pyplot
```
Comment (ilyasst) [inline requirements.txt]: The package in this case will be the whole `matplotlib`, we cannot only download/install the function `pyplot` from `matplotlib`. Could you change this line to `matplotlib` ?

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+PIL
```
Comment (ilyasst) [inline requirements.txt]: The package name is `Pillow` (https://pypi.org/project/Pillow/).

File: requirements.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+glob
```
Comment (ilyasst) [inline requirements.txt]: `glob` is included by default with python and does not require to be installed using pip or conda. Please remove it.

**Review by ilyasst — CHANGES_REQUESTED** [review]
Everything looks good except for `requirements.txt`; please consider the comments for that file and update the PR.

File: requirements.txt [hunk 7]
```diff
@@ -0,0 +1,5 @@
+pyyaml
+numpy
+matplotlib.pyplot
+PIL
+glob
```
Comment (ilyasst) [inline requirements.txt]: `glob` is included by default with python and does not require to be installed using pip or conda. Please remove it.

File: requirements.txt [hunk 8]
```diff
@@ -0,0 +1,5 @@
+pyyaml
+numpy
+matplotlib.pyplot
```
Comment (ilyasst) [inline requirements.txt]: The package in this case will be the whole matplotlib, we cannot only download/install the function pyplot from matplotlib. Could you change this line to matplotlib ?

**Review by ilyasst — CHANGES_REQUESTED** [review]
There are still some necessary changes for requirements.txt to run without errors. Please see the updated comments for requirements.txt

**Review by ilyasst — APPROVED** [review]
Everything is running fine in a fresh environment for me 👍

**Comment by ilyasst** [comment]
@arpcorrea Whenever you can, could you please add your review here by testing the code (running it locally on your machine). I think that we can merge after that.

**Review by arpcorrea — APPROVED** [review]
If possible, on the animation.py, try to include commentaries in some sections just for the better user understanding... for example:

#defining plot
plt.pcolormesh(x, y, z, vmin=self.matrix_generation.Troom, vmax=self.matrix_generation.Textrusion, cmap=cmapdeck)

#opening figures and making picture frames
for i in imgs:
            new_frame = Image.open(i)
            frames.append(new_frame)
            print(i)

**Pull request merged**